internal/dps/read.go

```diff
@@ -3,13 +3,13 @@ package dps
 import (
 	"context"
 	"encoding/json"
-	"errors"
 	"fmt"
 	"github.com/segmentio/kafka-go"
 )
 
-func ReadMessages(ctx context.Context, reader *kafka.Reader, callback func(*KafkaResponse, error)) error {
+func ReadMessages(ctx context.Context, reader *kafka.Reader, callback func(*KafkaResponse) error) error {
 	for {
+		// TODO log debug
```

Comment:
```suggestion
		// TODO(url/1234): use a proper logging library
```
Or perhaps delete the comment all together, but still create the logging library issue

Reply:
Comment removed. Issue #47 created.